sdk/core/Azure.Core/samples/Response.md

```diff
@@ -55,6 +55,18 @@ using (StreamReader reader = new StreamReader(contentStream))
 }
 ```
 
+## Accessing HTTP response content with `dynamic`
```

Comment:
Since this is likely to be the more common case as DPG takes over the world, should we consider reordering so that it is the first section on accessing HTTP content?

Reply:
Is that necessarily true given we want to move everyone to TypeSpec and, ergo, generate models? Perhaps, for now, this will become the primary case but that hopefully one day changes. I keep hearing different things from different people, though.

Reply:
I'll move it to before the GetRawResponse().ContentStream sample and after the Response\<T\> sample.